css/card.css

```diff
@@ -301,6 +301,13 @@ section#gacha-controls label {
 section#gacha-controls input {
   margin-left: 15px;
 }
+section#gacha-controls fieldset:nth-child(2){
```

Comment:
Do these CSS adjustments pertain to the animation changes? How, if so?

Reply:
The line that contains `fieldset:nth-child(2)` specifically is there to style the checkbox and separate it a bit further from the Card Pile/Card Grid radio buttons.